cfn-resources/federated-query-limit/cmd/resource/model.go

```diff
@@ -0,0 +1,17 @@
+// Code generated by 'cfn generate', changes will be undone by the next invocation. DO NOT EDIT.
+// Updates to this type are made my editing the schema file and executing the 'generate' command.
+package resource
+
+// Model is autogenerated from the json schema
+type Model struct {
+	ProjectId        *string `json:",omitempty"`
+	TenantName       *string `json:",omitempty"`
+	LimitName        *string `json:",omitempty"`
+	CurrentUsage     *string `json:",omitempty"`
+	DefaultLimit     *string `json:",omitempty"`
+	LastModifiedDate *string `json:",omitempty"`
+	MaximumLimit     *string `json:",omitempty"`
```

Comment:
im not able to find where we are using this property, if we do not need it lets remove ir because it creates confusion with the users

Reply:
These are optional read parameters, since nil check is handled, it shouldn't be a problem.